server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EchoMethodContract.scala

```diff
@@ -167,4 +154,15 @@ trait EchoMethodContract {
 
     assertThatJson(response).isEqualTo(RESPONSE_OBJECT_WITH_UNSUPPORTED_METHOD)
   }
+
+  private def getHeadersWith(authHeader: Header): Headers = {
+    new Headers(
+      new Header(ACCEPT.toString, Fixture.ACCEPT_RFC8621_VERSION_HEADER),
+      authHeader
+    )
+  }
+
+  private def toBase64(stringValue: String): String = {
```

Comment:
Should it be a utility method?

Reply:
It better in Fixture. agree with that

Reply:
May be we dont need that, it needed only for authentication test which only in `AuthenticationContract`.